distributed/batched.py

```diff
@@ -50,7 +49,6 @@ def __init__(self, interval, loop=None):
         self.batch_count = 0
         self.byte_count = 0
         self.next_deadline = None
-        self.recent_message_log = deque(maxlen=100)
```

Comment:
I am inclined to keep this functionality, but set `maxlen=` to some configuration option that defaults to zero

Reply:
Yes, I suppose it can be made configurable, as long as it's 0 by default ;-)